src/Population.py

```diff
@@ -51,14 +48,17 @@ def run(self):
             self.population = next_population
             self.observeEpoch()
 
+    def initialize(self):
+        self.population = self.generator.generate_population()
+
     def random_population_index(self):
         return random.randint(0, len(self.population) - 1)
 
     def generate(self, omitted):
         generated_indices = []
         for i in range(3):
             new_index = self.random_population_index()
-            while (generated_indices + omitted).count(new_index):
+            while generated_indices.count(new_index) or omitted == new_index:
```

Comment:
Okej, przyznaję, to, że przez chwilę miałam listę exceptions, w której to omitted było, zapędziło mnie w dziwne miejsce xd